MobileWallet/TariLib/Wrappers/Transactions/PendingInboundTransactions.swift

```diff
@@ -84,7 +84,7 @@ class PendingInboundTransactions: TransactionsProtocol {
             }
         }
 
-        let sortedList = list.sorted(by: { $0.localDate.0?.compare($1.localDate.0!) == .orderedDescending })
+        let sortedList = list.sorted(by: { $0.date.0?.compare($1.date.0!) == .orderedDescending })
```

Comment:
Its not ok to use !. maybe you check this also on UnitTesting?

Reply:
I wasn't sure how to handle this sorted callback with optional dates. I don't think a unit test can help as the Tari lib can cause an issue I'm not aware of. I'll make an issue to come back to and find a better way to handle optional dates.

Reply:
https://github.com/tari-project/wallet-ios/issues/new